android/demos/compose/src/main/kotlin/com/uber/rib/compose/root/RootView.kt

```diff
@@ -21,10 +21,12 @@ import android.util.AttributeSet
 import android.widget.FrameLayout
 import android.widget.TextView
 
-class RootView @JvmOverloads constructor(
+class RootView
+@JvmOverloads
+constructor(
```

Comment:
Previous form was superior IMO

Reply:
This change was done by `ktfmt`, we cannot customize this behavior but we can keep `ktfmt` disabled for now.